README.md

```diff
@@ -73,45 +73,73 @@ During a sprint we track progress using a [scrum or kanban-like board](#notes)<s
+
+A team should point each ticket as if any member were to take it on. Team members should therefore estimate each task as a group, and converge on a point value on which every member agrees. One method is planning poker, in which each team member reveals their individual point estimation of a task at the same time before the team collectively decides on the appropriate point value. 
+
+It does not matter if two teams use different pointing scales or other practices specific to their pointing process, as long as they are consistent within their own team. For example, a team might decide to always re-assess tickets that were assigned a point value of 13, because that suggests the task has not been understood and broken down sufficiently. 
+
 ## Velocity Tracking
 
-### What is Velocity Tracking?
+Velocity tracking uses a team's average historical output to inform how much work (measured in total number of points) to commit to in the next sprint. Velocity tracking can also help a team reflect on how much work was accomplished during retrospective by comparing their *estimated* sprint velocity to their *actual* sprint velocity, often [visualized with a burn-down chart](https://en.wikipedia.org/wiki/Burn_down_chart).
+
+### What is Sprint-Velocity?
+
+`Sprint-velocity` describes the rate at which work is completed over the duration of a sprint. In other words, sprint-velocity reflects the number of points completed by an individual team member in one hour (or points per person-hour).
 
-Let's say your team has broken user stories into tasks that are each ticketed, pointed, and assigned to team members. Now it's time to decide, how much should your team commit to as a whole for the next sprint? How does that translate to the total number of points that each team member should be assigned for the next sprint? Velocity tracking can help you do that by using a team's average historical output to inform how much work (measured in total number of points) to commit to in the upcoming sprint. As such, velocity tracking is both used to help us (1) plan upcoming sprints and (2) reflect on how much work was accomplished during retrospective.
+When using sprint-velocity to plan a sprint, a team typically wants to know its average velocity across fairly recent sprints to get an overall sense of a comfortable pace. We will refer to this later as the `average-sprint-velocity`.
+
+### How Do We Measure Sprint-Velocity?
+
+1. Determine the number of total available hours for the next 2-week sprint. If a team has 4 members, and each can contribute 40 hrs per week, there is a total of 320 person-hours. 
+
+ > ***total person-hours*** = The sum of the hours each team member can contribute to the sprint
+ 
+2. If a team is planning their first sprint and has never tracked velocity before, they should choose a "seed" sprint-velocity value, such as 0.5 points per person-hour, to get started. 
+
+ > ***sprint-velocity*** = number of points per person-hour
```

Comment:
This same variable is used below for a different value. This is really the average velocity that we'll be using for our estimate - but then you use that separately in step 5 to close the loop. Maybe we can call this one seed-sprint-velocity and then say in step 5 that the average-sprint-velocity will replace the seed-sprint-velocity for planning in the future (which it sort of does).

The one thing that's confusing is that there are two sprint-velocities.